modules/gapi/perf/common/gapi_core_perf_tests.cpp

```diff
@@ -0,0 +1 @@
+#include "gapi_core_perf_tests_inl.hpp"
```

Comment:
Please also put a license header in this file and include "test_precomp.hpp" (before any other includes)

Reply:
perf_precomp.hpp added